Fixed many games not working due to unsupported characters in file name.

JanErikGunnar opened the pull request.

JanErikGunnar committed:
Fixed many games not working due to unsupported characters in file name.

JanErikGunnar committed:
Fixed Mario Golf manual

JanErikGunnar committed:
Fixed import of some SMD save games (e.g. Monster World IV)

JanErikGunnar committed:
Improved import of some N64 roms (Ogre Battle 64 is no longer crashing)

JanErikGunnar committed:
Implemented export of FDS games (SMB: The lost levels etc)

JanErikGunnar committed:
Fixed export of many NES games (Double Dragon 2, Clu Clu Land, etc

JanErikGunnar committed:
Added .gitignore

JanErikGunnar committed:
Improved export of some filenames (e.g. Sin & Punishment, S.C.A.T., etc)

JanErikGunnar committed:
Minor documentation

JanErikGunnar committed:
Implemented extracting FDS BIOS (not executed yet)

JanErikGunnar committed:
Exported NES files are no longer larger than needed

JanErikGunnar committed:
Extracting NeoGeo games, but not uncompressed

JanErikGunnar committed:
Decompressing zlib compressed NeoGeo games

JanErikGunnar committed:
Extracting first Neo Geo game: Neo Turf Masters

JanErikGunnar committed:
Exporting Neo Geo game Kong of the Monsters

JanErikGunnar committed:
Exporting Neo Geo game Magician Lord

JanErikGunnar committed:
Better error handling of Neo Geo game.bin.xz files

JanErikGunnar committed:
Exporting Neo Geo game Metal Slug 2

JanErikGunnar committed:
Exporting Neo Geo game Metal Slug

JanErikGunnar committed:
Neo Geo: Reduced complexity of game specific code

JanErikGunnar committed:
Exporting Neo Geo game Magical Drop 3

JanErikGunnar committed:
Exporting Neo Geo game Spin Master

JanErikGunnar committed:
Neo Geo export no longer requires hardcoded NGH-numbers

JanErikGunnar committed:
Neo Geo: minor refactoring

JanErikGunnar committed:
Neo Geo: Wii game ID no longer used, will attempt to export more games

JanErikGunnar committed:
Neo Geo: Minor refactoring

JanErikGunnar committed:
Neo Geo: Refactoring (less game specific code)

JanErikGunnar committed:
Neo Geo: Extracted BIOS has mame-compatible file name

JanErikGunnar committed:
Neo Geo: Minor refactoring

JanErikGunnar committed:
Neo Geo: Minor refactoring

JanErikGunnar committed:
Neo Geo: Exporting save data

JanErikGunnar committed:
Neo Geo: Minor documentation

JanErikGunnar committed:
Neo Geo: Extracting unexpected ROMs, if any

JanErikGunnar committed:
Neo Geo: Minor documentation

JanErikGunnar committed:
Fixed export of Mario Golf 64 manual missing some data

JanErikGunnar committed:
Updated Readme

JanErikGunnar committed:
Correct spelling of TurboGrafx in readme :)

JanErikGunnar committed:
Can export TurboGrafx CD games

JanErikGunnar committed:
Minor: Fixed an incorrect comment

JanErikGunnar committed:
Removed output of some debug files for TGCD

JanErikGunnar committed:
Updated Readme

JanErikGunnar committed:
SMS saves and additional Genesis save games extracted.

JanErikGunnar committed:
Minor documentation

JanErikGunnar committed:
FDS save games are exported

JanErikGunnar committed:
Added broken Shadow of the Ninja to known issues

JanErikGunnar committed:
Minor Neo Geo improvements

JanErikGunnar committed:
Partially extracting Space Harrier (arcade)

JanErikGunnar committed:
Minor progress on extracting Space Harrier and Ghosts n Goblins (arcade)

JanErikGunnar committed:
Fixed title of games with roman numeral "I" (such as "Ys Book I & II")

treymerkley committed:
Fix crash on 256*1024 files

treymerkley committed:
fix crash on 256*1024 file size

treymerkley committed:
fix unboundexception

treymerkley committed:
Merge pull request #1 from JanErikGunnar/master

merge from recent

treymerkley committed:
Prevent total failure if NES save file recovery breaks

JanErikGunnar committed:
Merge pull request #1 from billywade/master

little fixes to make it run more smoothly

JanErikGunnar committed:
Update about known N64 issues

JanErikGunnar committed:
Merge branch 'master' of https://github.com/JanErikGunnar/vcromclaim

JanErikGunnar committed:
Fixed Ogre Battle 64 and (partially) Mario Tennis.

JanErikGunnar committed:
Updated readme regarding Ogre Battle 64 and Mario Tennis

Plombo commented:
Hi! I see you've been maintaining your fork for quite a while. Thanks for caring for it, and apologies for the low quality of my original code.

I know it's quite late for this, but would you like push access to this repository, or would you rather just keep on with your fork at this point?

treymerkley commented:
I'd be happy to take push access, but I only really need it once, so I don't really have any other commits.

JanErikGunnar committed:
Made readme applicable to both Plombo and JanErikGunnars fork.

JanErikGunnar commented:
Hi!
Nothing to apologize for! And I should thank you for the huge effort in creating this to begin with, clearly it took a lot of work and it's been very useful for me to liberate my VC library :) Besides, my additions is probably not fantastic either, I barely know Python so one of the reasons for doing it has been to learn it a bit better.

I have few more things I would like to implement, I don't mind continuing to do it in my fork but I'm happy to push merges of it to your repo if you think the changes are worthwhile.

JanErikGunnar committed:
Extract or create all BIOS ROMs for NeoGeo games

JanErikGunnar committed:
Improved readme and some minor Neo Geo documentation

JanErikGunnar committed:
Merge branch 'master' of https://github.com/JanErikGunnar/vcromclaim

JanErikGunnar committed:
Group files by system, then game

Fixes problems when there are games for
different systems with the same name

JanErikGunnar committed:
Arcade: Fix bug on Space Harrier, minor progress on Ghost n Goblines (Arc)

JanErikGunnar committed:
Fix several broken SNES games, and add arcade Ghosts n Goblins manual

Correctly extract SNES games that was broken in a previous version: Super Mario RPG, Secret of Mana, Kirbys Dream Land 3, Super Punch-out, probably more

Extract Arcade Ghosts n Goblins manual (maybe also other Arcade manuals)

Refactor LZ77 code (questinable result...)

JanErikGunnar committed:
Extracting Space Harrier (Arcade) gfx1 roms

JanErikGunnar committed:
Extract decrypted Neo Geo games!

JanErikGunnar committed:
Add credits

JanErikGunnar committed:
Minor fix in readme

JanErikGunnar committed:
Minor progress on exporting Metal Slug 3 (NeoGeo)

JanErikGunnar committed:
Minor progress on Metal Slug X (NeoGeo)

JanErikGunnar committed:
Minor progress on Metal Slug 4 (NeoGeo)

JanErikGunnar committed:
Can extract NAM-1975 (Neo Geo)

JanErikGunnar committed:
Extract Real Bout Fatal Fury Special (Neo Geo)

JanErikGunnar committed:
Extract Shock Troopers (Neo Geo)

JanErikGunnar committed:
Progress on extracting The Last Blade 2 (Neo Geo)

JanErikGunnar committed:
Extract The Last Blade 2 (Neo Geo)

Implement decompression of ACM compressed C ROMs
Extract The Last Blade 2
Minor progression on Shock Troopers 2

JanErikGunnar committed:
Extract Shock Troopers 2 (Neo Geo)

Also some progress on Metal Slug 3 and on generic Neo Geo guess.

JanErikGunnar committed:
Minor progress on Metal Slug X, 3 and 4

The C roms are not ACM compressed anymore,
but they are decrypted and we need to encrypt them
again to get playable in mame.

JanErikGunnar committed:
Extract Metal Slug X (Neo Geo)

JanErikGunnar committed:
Minor Neo Geo quality of life improvements

- Exporting ACM compressed NG games is faster
- Storing Neo Geo save games in correct folder

JanErikGunnar committed:
NeoGeo improvements

- Fix broken ACM CROMs
- Extract patched versions of system ROM
- Minor fixes

JanErikGunnar committed:
Upgrade to Python 3.8

JanErikGunnar committed:
Properly export Mario Tennis (N64)

JanErikGunnar committed:
Extract King of Fighters 98 (NG)

JanErikGunnar committed:
Extract Space Harrier (Arcade)

JanErikGunnar committed:
Fix crash when encountering encrypted Neo Geo games with missing key

JanErikGunnar committed:
Add documentation about Super Air Zonk

JanErikGunnar committed:
Minor documentation

JanErikGunnar committed:
Fix regression of many SNES games and manuals

Many SNES games, SNES manuals and NES manuals got broken when migrating to python 3.

JanErikGunnar committed:
Extract TG save files

JanErikGunnar committed:
Fix docs (XZ CAN be extracted); more clarifications

JanErikGunnar committed:
Metal Slug 4 (NG): Extract C+S ROMs

Only the M ROM now remains for the extracted game to be playable

JanErikGunnar committed:
NES games with SRAM extracted better

At least Kirby's Adventure, possibly other games, is now extracted
properly with or without save data.
Before this commit, it would crash if there is save data, or export a
ROM with the SRAM flag OFF if there is no save data.

JanErikGunnar committed:
Fix Mario Golf (maybe others) N64 save games

badlydrawnface committed:
use time.process_time() instead of time.clock()

time.clock() was removed in 3.8, and I had to change it to decompress an snes rom from Kirby's Dream Collection

JanErikGunnar committed:
Merge pull request #5 from badlydrawnface/master

use `time.process_time()` instead of `time.clock()`

JanErikGunnar committed:
Minor code cleanup, update Python requirement

JanErikGunnar committed:
Fix typos in readme

JanErikGunnar committed:
Metal Slug 4 (NG): Extract M ROM

JanErikGunnar committed:
NG QoL improvements

JanErikGunnar committed:
Metal Slug 4 (NG) fully exported

JanErikGunnar committed:
Minor fixes

JanErikGunnar committed:
NG: Slight ACM performance optimization, maybe...

JanErikGunnar committed:
Metal Slug 3 (NG): Extract P ROM

JanErikGunnar committed:
Metal Slug 3 (NG) fully extracted

JanErikGunnar committed:
Minor Readme fixes

JanErikGunnar committed:
Ghosts'n Goblins (Arcade) fully extracted

JanErikGunnar committed:
Exract all NG without manually supplying AES key

JanErikGunnar committed:
Minor documentation updates

JanErikGunnar committed:
Minor documentation

JanErikGunnar committed:
Spelling mistake in README.md

alhumbra committed:
kof94 and karnovr fully extracted

JanErikGunnar committed:
Merge pull request #6 from alhumbra/master

kof94 and karnovr fully extracted

JanErikGunnar committed:
Neo Geo decryption simplified
